client/app/bundles/KudosApp/components/GiveKudo.jsx

```diff
@@ -51,11 +30,11 @@ export class GiveKudo extends React.Component {
     // Uses lodash to bind all methods to the context of the object instance, otherwise
     // the methods defined here would not refer to the component's class, not the component
     // instance itself.
-    _.bindAll(this, 'handleClick', 'onChangeSearchInput', 'setMessage')
+    _.bindAll(this, 'handleClick', 'onSelectChange', 'setMessage', 'onSuccess', 'onFailure')
```

Comment:
Interesting, not sure why this worked before this PR, since you added the `_` lodash import.  But let's make it consistent with the other imports: you can refer here to `bindAll`, and then import `bindAll` in the spread on line 1, along with `isEmpty` and `trim`.